flow360/examples/actuatorDisk/flow360.json

```diff
@@ -25,7 +25,7 @@
       "startAverageIntegrationStep": -1
     },
     "surfaceOutput": {
-      "outputFormat": "both",
+      "outputFormat": "tecplot",
```

Comment:
why are these changed?

Reply:
when I tried to run this example, it failed saying "both" is not supported